src/Pickles/Pickles.DocumentationBuilders.Html/HtmlImageResultFormatter.cs

```
@@ -153,7 +153,13 @@ public XElement Format(ScenarioOutline scenarioOutline, params string[] exampleV
         {
             if (this.configuration.HasTestResults)
             {
-                TestResult exampleResult = this.results.GetExampleResult(scenarioOutline, exampleValues);
+                TestResult exampleResult = TestResult.Inconclusive;
+
+                try
+                {
+                    exampleResult = this.results.GetExampleResult(scenarioOutline, exampleValues);
+                }
+                catch (Exception) { }
```

You're swallowing an exception without handling it in any way. That's bad.